lib/pymedphys/_dicom/connect/listen.py

```diff
@@ -106,6 +103,16 @@ def on_association_released(self, _):
         if self.on_released_callback:
             self.on_released_callback(self.association_directory)
 
+    def _build_hierarchical_path_to_series(
+        self, test_dataset: "pydicom.dataset.Dataset"
+    ) -> pathlib.Path:
+        series_path = pathlib.Path(self.storage_directory).joinpath(
+            test_dataset.PatientID,
+            test_dataset.StudyInstanceUID,
+            test_dataset.SeriesInstanceUID,
+        )
+        return series_path
```

Comment:
Nit.

Is this function better separated from out of the class? It seems like it might be generally useful as a part of a "DICOM storage" part of PyMedPhys. A standalone function that receives a directory and a DICOM file and it spits out the PATH to where that DICOM file should be found/placed.

Reply:
This can be addressed when this is refactored out in the future, but perhaps rename the variable from `test_dataset` to `dicom_dataset` or `dataset`.